api/config/urls.py

```diff
@@ -50,6 +53,7 @@
     path('api/auth/', auth_urls),
     path('api/', meetup_urls),
     path('api/meetup/', question_urls),
+    path('api/', answer_urls),
```

Comment:
The URL should be `POST /api/meetups/{meetupId}/questions/{questionId}/answers`

This will make sure you have the right meetup first. Then the right question, so that it can be answered. Remember questions are specific to meetups. And answers are specific to questions.

Reply:
Okay, making the changes